alpha/external-web/app/eligibility-client.js

```diff
@@ -47,6 +47,25 @@ module.exports = {
         callback(error, null)
       }
     })
+  },
+
+  /**
+   * Update an existing claimant.
+   * @param id The id of the claimant to update.
+   * @param sectionName The name of the embedded section to update.
+   * @param section The section details to use for the update.
+   * @param callback A callback defining what to do after a successful and failed update.
+   */
+  embeddedUpdate: function (id, sectionName, section, callback) {
```

Comment:
You method updates an embedded field, it doesn't care if the value being used to update is a sub-document (section in your context), it could be re-used later to set a single value on the document such as status.

Rename to UpdateField and rename params to fieldName and data to be clear